src/ResourceManager/Profile/Commands.Profile/help/Add-AzureRmEnvironment.md

```diff
@@ -434,61 +480,31 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -TrafficManagerDnsSuffix
-Specifies the domain-name suffix for Azure Traffic Manager services.
-
-```yaml
-Type: String
-Parameter Sets: Name
-Aliases: 
-
-Required: False
-Position: 12
-Default value: None
-Accept pipeline input: True (ByPropertyName)
-Accept wildcard characters: False
-```
-
-### -BatchEndpointResourceId
-The resource identifier of the Azure Batch service that is the recipient of the requested token
+### -StorageOAuthEndpointResourceId
+The audience for tokens authenticating with Azure Storage.
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases: 
 
 Required: False
-Position: 20
+Position: 23
```

Comment:
this is a non mandatory param, kindly remove the position requirement on it.

Reply:
fixed